user_tools/docs/user-tools-onprem.md

```diff
@@ -1,7 +1,9 @@
 # RAPIDS User Tools on OnPrem platform
 
 This is a guide for the RAPIDS tools for Apache Spark clusters that are provisioned manually (a.k.a onPrem). At the end of this guide, the user will be able to run the RAPIDS tools to analyze the clusters and the applications running OnPrem.
-
+RAPIDS tools to analyze the clusters and the applications running OnPrem. <Space><Space>
```

Comment:
Instead of "switch the worker node instance types", you can say "please use these worker node instance types" since they are migrating to a new platform.

Reply:
Fine, except change `OnPrem` to `on-premises`.

Reply:
@mattahrens Do you want to use "on-premises"  throughout the entire documentation, or only in the above line

Reply:
For documentation, we can switch to "on-premises" throughout the entire doc.  But the command short name can still be "onprem".